packages/graph/hash_graph/lib/graph/src/types/mod.rs

```diff
@@ -1 +1,90 @@
+//! Model types used across datastores
```

Comment:
Not sure what happened in this, but the file at
https://github.com/hashintel/hash/blob/ec1a2fb665500256aa77b31bfc3bc0fb788835b1/packages/graph/hash_graph/lib/graph/src/ontology/mod.rs contains some of the same defs, but not the qualified

Reply:
Definitely, my IDE tried to be smart 🙃